tracker/corpus.py

```diff
@@ -0,0 +1,84 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import bz2
+import json
+import logging
+import os
+import pathlib
+
+from esrally.utils import console
+
+
+def template_vars(index_name, out_path, comp_outpath, doc_count):
+    corpus_path = pathlib.Path(out_path)
+    compressed_corpus_path = pathlib.Path(comp_outpath)
+    return {
+        "index_name": index_name,
+        "base_url": corpus_path.parent.as_uri(),
+        "filename": corpus_path.name,
+        "path": corpus_path,
+        "doc_count": doc_count,
+        "uncompressed_bytes": os.stat(corpus_path.as_posix()).st_size,
```

Comment:
Can we use `os.path.getsize` instead here?

Reply:
https://github.com/elastic/rally/pull/875/commits/0848be0aef5a3232af0078d456b1cbd823585f27